packages/vuepress-theme-odyssey/styles/base/_fonts.scss

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020-Present Okta, Inc.
```

Comment:
Please go through all these files to make sure we have legal headers. We're missing them in a few places!

Reply:
[Here](https://github.com/okta/odyssey/pull/728#discussion_r511064468) you wrote:

> Since this isn’t being publicly distributed, we don’t need to add the license files here.

Which I took to mean remove the copyright notice. Did I misinterpret it?

Reply:
Spoke to Jordan offline, is looking for consistency. Notice has been added back in to all files.